js/widgets/fixedToolbar.js

```diff
@@ -37,22 +39,23 @@
 
 return $.widget( "mobile.toolbar", $.mobile.toolbar, {
 	options: {
-		position: null,
+		position:null,
 		visibleOnPageShow: true,
 		disablePageZoom: true,
-		transition: "slide", //can be none, fade, slide (slide maps to slideup or slidedown)
+
+		// Can be none, fade, slide (slide maps to slideup or slidedown)
+		transition: "slide",
 		fullscreen: false,
-		tapToggle: true,
-		tapToggleBlacklist: "a, button, input, select, textarea, .ui-header-fixed, .ui-footer-fixed, .ui-flipswitch, .ui-popup, .ui-panel, .ui-panel-dismiss-open",
 		updatePagePadding: true,
-		trackPersistentToolbars: true,
 
 		// Browser detection! Weeee, here we go...
-		// Unfortunately, position:fixed is costly, not to mention probably impossible, to feature-detect accurately.
-		// Some tests exist, but they currently return false results in critical devices and browsers, which could lead to a broken experience.
-		// Testing fixed positioning is also pretty obtrusive to page load, requiring injected elements and scrolling the window
-		// The following function serves to rule out some popular browsers with known fixed-positioning issues
-		// This is a plugin option like any other, so feel free to improve or overwrite it
+		// Unfortunately, position:fixed is costly, not to mention probably impossible,
+		// to feature-detect accurately. Some tests exist, but they currently return false results
+		// in critical devices and browsers, which could lead to a broken experience. Testing fixed
+		// positioning is also pretty obtrusive to page load, requiring injected elements and
+		// scrolling the window. The following function serves to rule out some popular browsers
+		// with known fixed-positioning issues. This is a plugin option like any other, so feel
+		// free to improve or overwrite it
 		supportBlacklist: function() {
```

Comment:
remove the support blacklist we dont support any of these browsers any more. lets remove this test from support and make this function a noop and move it to backcompat